backtracking/minimax.cpp

```diff
@@ -1,27 +1,42 @@
+/**
+ * @file
+ * @brief returns which is the longest/shortest number, using minimax algorithm
+ *
+ * Wikipedia: https://en.wikipedia.org/wiki/Minimax
```

Comment:
The way we are adding references is by using Markdown syntax. See for example other files [like this](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/numerical_methods/bisection_method.cpp) in the repo.